api/integrations/slack/views.py

```diff
@@ -19,40 +21,45 @@
 from integrations.slack.slack import SlackWrapper
 
 from .authentication import OauthInitAuthentication
-from .exceptions import FrontEndRedirectURLNotFound, InvalidStateError
+from .exceptions import (
+    FrontEndRedirectURLNotFound,
+    InvalidStateError,
+    SlackConfigurationDoesNotExist,
+)
+from .pagination import ChannelListPagination
 from .permissions import OauthInitPermission
 
 signer = TimestampSigner()
 
 
+class SlackGetChannelsViewSet(GenericViewSet):
+    serializer_class = SlackChannelListSerializer
+    pagination_class = ChannelListPagination
+
+    def get_queryset(self) -> SlackWrapper:
```

Comment:
Hmm... I'm not sure about the fact that we're returning a `SlackWrapper` object here? The docs for `get_queryset` state: 

```
        Get the list of items for this view.
        This must be an iterable, and may be a queryset.
        Defaults to using `self.queryset`.
```

Reply:
get_queryset -> get_slack_wrapper ?